GSA/made-in-america
# updated with additional urgent waiver

Post Urgent Waivers to transparency site

How to test
Pull this specific branch and start the project locally.
Navigate to /about/urgent-waivers and check file downloads and make sure that each file is the same as the file listed in the ticket.

Navigate to the public website and validate the spreadsheet is visible and the user is able to download it.
see example:

https://www.madeinamerica.gov/about/urgent-waivers/

Ticket: https://cm-jira.usa.gov/browse/MIA-358
